setup.py

```diff
@@ -224,10 +224,7 @@ def initialize_options(self) -> None:
 ]
 
 vertica_dependencies = [
-    # sqlalchemy-vertica-python 0.5.5 introduced
-    # https://github.com/bluelabsio/sqlalchemy-vertica-python/pull/7
-    # which fixed a bug pulling schema information from Vertica
-    'sqlalchemy-vertica-python>=0.5.5,<0.6',
+    'sqlalchemy-vertica-python',
```

Comment:
do we still want to keep the min version?

Reply:
reinstated min version